src/atoms/TextInput/TextInput.native.js

```diff
@@ -200,8 +222,14 @@ const TextInput = ({
         <Flex flexDirection="row" alignItems="flex-start">
           <View>
             {/* Fixed Left Label */}
-            {hasLeftLabel && layoutDimensions ? (
-              <Label.Regular inputLayoutDimensions={layoutDimensions}>{label}</Label.Regular>
+            {!hasAnimatedLabel && layoutDimensions && labelPosition === 'left' ? (
```

Comment:
just `labelPosition === 'left'` would work?

Reply:
We need a check for ```layoutDimensions``` and ```!hasAnimatedLabel``` is needed to ensure that we don't render this label for position left & outlined variant